packages/core/src/components/BaseInput/BaseInput.styles.tsx

```diff
@@ -159,17 +158,17 @@ export const { staticClasses, useClasses } = createClasses("HvBaseInput", {
       background: theme.colors.atmo1,
     },
 
-    "& $inputRootReadOnly": {
-      backgroundColor: "transparent",
+    "&$inputRootReadOnly": {
+      backgroundColor: theme.colors.atmo3,
     },
   },
   inputRootDisabled: {
-    background: theme.colors.atmo2,
+    background: theme.colors.atmo3,
```

Comment:
maybe these could be set in the parent. fewer overrides, easier to change

```suggestion
    background: "none",
```

Reply:
I agree, I'll look into this!

Reply:
I'll revert back the background color. It will most likely be `atmo3` in the future but DS is still making some tests. Once it's official we'll change it.